src/Button.js

```diff
@@ -10,7 +10,7 @@ const propTypes = {
   disabled: PropTypes.bool,
   outline: PropTypes.bool,
   tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
-  getRef: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  innerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
```

Comment:
Curious ... I know `getRef()` was used in a lot of places. Are we switching to `innerRef()` ?  It actually makes more sense intuitively as I ran into this issue during development and forgot the actual `ref` was `getRef()`.

Reply:
This was to line up with other libraries. I considered keeping both, aliasing one to the other but thought doing so might cause some complications.

Reply:
Cool, let's just use `innerRef` and make sure all other components use that.